src/utils/hooks/useCreateMakeDecisionReview.ts

```diff
@@ -24,48 +18,41 @@ type ConstructReviewPatch = (structure: FullStructure) => ReviewPatch
     reviewStructure,
     reviewAssignment,
-    previousAssignment,
   })
 
-  const shouldRemakeFinalDecisionReviewResponse = (element: PageElement) => {
-    if (previousAssignment?.level === 1) return true
-    return element?.thisReviewLatestResponse?.decision === ReviewResponseDecision.Agree
-  }
-
   const constructReviewPatch: ConstructReviewPatch = (structure) => {
     const elements = Object.values(structure?.elementsById || {})
 
     // Exclude not assigned, not visible and missing responses
     const reviewableElements = elements.filter((element) => {
       const { isAssigned, isActiveReviewResponse } = element
-      return (
-        shouldRemakeFinalDecisionReviewResponse(element) && isAssigned && !isActiveReviewResponse
-      )
+      return isAssigned && !isActiveReviewResponse
     })
 
-    // For re-assignment this would be slightly different, we need to consider latest review response of this level
-    // not necessarily this thisReviewLatestResponse (would be just latestReviewResponse, from all reviews at this level)
+    // Generate each reviewResponse for a MakeDecision review as a copy of previous Lower Level review.
+    // The lowerLevelReview responses are mapped to latestOriginalReviewResponse in helpers
+    // addAllReviewResponses function after being stored on query getReviewResponses under
+    // previousOriginalReviewResponses (not very easy to track - so commented here for reference!)
     const reviewResponseCreate = reviewableElements.map(
-      ({ isPendingReview, thisReviewLatestResponse, response, lowerLevelReviewLatestResponse }) => {
-        const applicationResponseId =
-          !previousAssignment || previousAssignment.level > 1 ? undefined : response?.id
+      ({ response, latestOriginalReviewResponse, thisReviewLatestResponse }) => {
+        const applicationResponseId = response?.id
         const reviewResponseLinkId =
-          !previousAssignment || previousAssignment.level === 1
-            ? undefined
-            : lowerLevelReviewLatestResponse?.id
-        // create new if element is awaiting review
-        const shouldCreateNew = isPendingReview
+          latestOriginalReviewResponse?.reviewResponseLinkId ??
+          thisReviewLatestResponse?.reviewResponseLinkId
         return {
           // Create new decision and comment if lower level review response or application was change, otherwise duplicate previous review response
-          decision: shouldCreateNew ? null : thisReviewLatestResponse?.decision,
-          comment: shouldCreateNew ? null : thisReviewLatestResponse?.comment,
+          decision: latestOriginalReviewResponse?.decision
+            ? latestOriginalReviewResponse?.decision
+            : thisReviewLatestResponse?.decision ?? null,
+          comment: latestOriginalReviewResponse?.decision
+            ? latestOriginalReviewResponse?.comment
+            : thisReviewLatestResponse?.comment ?? null,
```

Comment:
This is the fix required to get the previous review_response decision showing on Final decision - instead of creating new ones as before.